### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by bellisk**

### CODE REVIEW

File: source/content/glossar/bibliothek/dcat-examples/dataset-publisher-rdf.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .. code-block:: xml
-    :caption: Publisher of a Dataset current implementation
+    :caption: Publisher of a dataset: current implementation in RDF
```
Comment (sabinem) [inline source/content/glossar/bibliothek/dcat-examples/dataset-publisher-rdf.rst:2]: It think we should remove the word "current" here: "Publisher of a dataset in RDF"

File: source/content/glossar/bibliothek/dcat-examples/dataset-publisher-ttl.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .. code-block:: Turtle
-    :caption: Publisher of a Dataset current implementation in Turtle: does not conform to DCAT-AP-CH
+    :caption: Publisher of a dataset: current implementation in Turtle
```
Comment (sabinem) [inline source/content/glossar/bibliothek/dcat-examples/dataset-publisher-ttl.rst:2]: Same as above: "Publisher of a dataset in Turtle"

File: source/content/glossar/bibliothek/dcat-examples/dataset-publisher-rdf.rst (hunk 1 above)
Comment (bellisk) [reply source/content/glossar/bibliothek/dcat-examples/dataset-publisher-rdf.rst]: I agree, removed it.

File: source/content/glossar/bibliothek/dcat-examples/dataset-publisher-ttl.rst (hunk 2 above)
Comment (bellisk) [reply source/content/glossar/bibliothek/dcat-examples/dataset-publisher-ttl.rst]: I agree, removed this too.

**Pull request merged**